### DISCUSSION TITLE
Daily standup

### DISCUSSION THREAD

**Discussion by Arlet04** [SistemasTecTlaxiaco/webfood#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Arlet04** [comment]
*Responden a las siguientes preguntas de manera breve:
- ¿Qué hice en la última sesión?
- ¿Qué haré en esta sesión?
- ¿Qué obstáculos tengo?

**Reply by lucero012** [reply]
1.- Se creó el sprint planning
2.- Avanzar con las actividades
3.- El internet

**Reply by Nelsy21** [reply]
1. Se realizó el sprint planing de las actividades propuestas para la unidad número 3. 
2. La organización y repartción de roles y  de actividades a cada integrante del equipo de desarrollo
3. Internet

**Reply by Anamichel-leon** [reply]
1. Se realizo en la sección anterior el sprint en Kanban y en el road map se definieron fechas de entrega 
2. Se implementara un contador y posteriormente una wallet al contrato inteligente echo en Rust en el Sprint anterior
3. Las versiones de los paquetes instalados anteriormente y el internet inestable
